schema/deploy/table_user.sql

```diff
@@ -42,11 +42,11 @@ comment on column ggircs_portal.user.created_by is 'The person who updated the u
 comment on column ggircs_portal.user.updated_at is 'The date the user was updated';
 comment on column ggircs_portal.user.updated_by is 'The person who updated the user';
 
-insert into ggircs_portal.user (id, uuid, first_name, last_name, email_address)
-        values (1, '6c01258f-6ad8-4790-8ccc-485163f122a5' , 'Hamza', 'Javed', 'hamza@button.is');
-insert into ggircs_portal.user (id, uuid, first_name, last_name, email_address)
-        values (2, 'ca716545-a8d3-4034-819c-5e45b0e775c9' , 'Alec', 'Wenzowski', 'alec@button.is');
-insert into ggircs_portal.user (id, uuid, first_name, last_name, email_address)
-        values (3, '11bc5fb8-88f6-4b6e-92e3-581622ad4804', 'Kawhi', 'Leonard', 'kawhi@button.is');
+insert into ggircs_portal.user (uuid, first_name, last_name, email_address)
+        values ('6c01258f-6ad8-4790-8ccc-485163f122a5' , 'Hamza', 'Javed', 'hamza@button.is');
+insert into ggircs_portal.user (uuid, first_name, last_name, email_address)
+        values ('ca716545-a8d3-4034-819c-5e45b0e775c9' , 'Alec', 'Wenzowski', 'alec@button.is');
+insert into ggircs_portal.user (uuid, first_name, last_name, email_address)
+        values ('11bc5fb8-88f6-4b6e-92e3-581622ad4804', 'Kawhi', 'Leonard', 'kawhi@button.is');
```

Comment:
The update from `id` to `uuid` makes sense...but do we even need to be inserting dummy data here? It makes me uncomfortable, especially since this is all supposedly coming from the `session` in #88 and should be imported on registration by this PR

Reply:
FYI, those inserts are being removed in the `feature/data-initialization-script` branch, so don't worry about those

Reply:
Can you undo these changes to prevent future conflicts?

Reply:
Removed inserts from table_user